doc/part-1-github-ui/github-contributions.md

```diff
@@ -8,6 +8,41 @@ description: 'PART 1: GITHUB UI'
 
 At this point, the focus is not about Git and Github best practices. The learning videos here will cover the basic things needed to get started with contributing to any project using the Github UI. What you will learn will also help you to tackle the tasks that have been provided for you to do after watching the learning videos.
 
+### You will learn the following in this section:
+
+1. **Forking a repository.**
+   - Know basic settings while forking a repository.
+   - Know which particular branch you are in when you fork a repository.
+2. **Contributing to a repository.**
+   - Overview on contributions to a repository. 
```

Comment:
Wow, nice @aaryansr26 
You can really structure content. This is a syllabus. I like it. I'm still reviewing though.
You watched the youtube videos right?

Reply:
Yes ma'am, I went through all the videos. But I have not gone through with the last video in which the tasks were assigned (Task 1 and Task 2), do you want me to add that video also to the list? @Ifycode

Reply:
Nice. 
Ok no, don't bother adding the task videos @aaryansr26

#
#### Change request 1
Help change the title of your pull request following what is written here:https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/making-and-submitting-changes
It helps me easily know the issue the pull request fixes without having to open a pull request.

Reply:
I really love the structure you gave it. And it gives me a whole new whole new idea of how I can do things better. But I have a few things to say:
1. This project will be removed from my personal repo, and will continue to be developed in the @code-collabo community as from April 1st, 2023. Here is the new repo [@code-collabo/git-github-collabo-course](https://github.com/code-collabo/git-github-collabo-course)
2. There is ongoing plan to structure change a lot of things and finish up the unfinished project. So there will be a re-do of all the existing videos. The redo is because we want to fix a lot of wordings, inconsistencies, repetitions that comes along with making spontaneous videos etc.

That said, I will drop a few more change requests in a few minutes. 

@aaryansr26

Reply:
I have changed the title! @Ifycode

Reply:
I have changed the title as requested! @Ifycode

Reply:
👍🏼